stdlib/fun.mli

```diff
@@ -31,3 +31,31 @@ val flip : ('a -> 'b -> 'c) -> ('b -> 'a -> 'c)
 val negate : ('a -> bool) -> ('a -> bool)
 (** [negate p] is the negation of the predicate function [p]. For any
     argument [x], [(negate p) x] is [not (p x)]. *)
+
+val protect : acquire:(unit -> 'a) -> release:('a -> unit) -> ('a -> 'b) -> 'b
+(** [protect ~acquire ~release work] invokes [acquires], then [work],
+   and then invokes [release] when [work] returns with its value or an
+   exception. In the latter case the exception is re-raised after
+   [release]. If [release] raises an exception, then the exception
+   [Release_failure] is raised instead, as documented below.
```

Comment:
Where is that "as documented below" supposed to refer to ?

Reply:
I have rephrased.